gravitee-am-ui/src/app/components/gio-license/gio-license.service.ts

```diff
@@ -149,11 +153,40 @@ const featureMoreInformationData = {
   },
 };
 
+export type License = {
+  tier: string;
+  packs: Array<string>;
+  features: Array<string>;
+};
+
+export type Feature = {
+  feature?: string;
+  deployed?: boolean;
+}
+
 @Injectable({
   providedIn: 'root',
 })
 export class GioLicenseService {
 
+  constructor(private readonly http: HttpClient) {}
+
+  private loadLicense$: Observable<License> = this.http.get<License>(`${AppConfig.settings.baseURL}/platform/license`).pipe(shareReplay(1));
+
+  getLicense$(): Observable<License> {
+    return this.loadLicense$;
+  }
+
+  isMissingFeature$(feature: Feature): Observable<boolean> {
+    if (feature == null || feature.deployed === true) {
+      return of(false);
+    }
+    if (feature.deployed === false) {
+      return of(true);
+    }
+    return this.getLicense$().pipe(map((license) => license == null || license.features.find((feat) => feat === feature) == null));
```

Comment:
```suggestion
    return this.getLicense$().pipe(any((license) => license == null || license.features.includes(feature));
```
Does it make the same things ?

Reply:
> Does it make the same things ?

No. but this yes :
```
return this.getLicense$().pipe(any((license) => license == null || !license.features.includes(feature));
```

Reply:
I think that `any` and `includes` is easier to read. Nevertheless, it works for both, so no issue 😊